cas-server-documentation/integration/Delegate-Authentication.md

```diff
@@ -153,7 +87,7 @@ To add SAML support, add the following dependency:
 
 A provider is a server which can authenticate user (like Google, Yahoo...) instead of a CAS server. If you want to delegate the CAS authentication to Twitter for example, you have to add an OAuth client for the provider: Twitter. Clients classes are defined in the pac4j library.
 
-All the needed clients to authenticate against providers must be declared in the `applicationContext.xml` file:
+All the needed clients to authenticate against providers must be declared in a specific `WEB-INF/spring-configuration/pac4jContext.xml` file:
 
 {% highlight xml %}
 <bean id="facebook1" class="org.pac4j.oauth.client.FacebookClient">
```

Comment:
I am not sure. The most popular uses cases FB, OIDC and SAML. You are at the most going to have 5 clients in practice. While in theory it's possible to create as many clients as you want, in practice it will never be more than 4-5, and there wont be duplicate clients (i.e. no two facebook clients). 

I think we should shoot for the simple case and automate it like that. if anyone wants to declare 20 clients, that's edge case that could happen via explicit xml config